README.md

```diff
@@ -0,0 +1,262 @@
+community.
+
+[open-source contributions]: https://github.com/quantumlib/Cirq/graphs/contributors
+[code of conduct]: https://github.com/quantumlib/cirq/blob/main/CODE_OF_CONDUCT.md
+
+### Announcements
+
+Stay on top of Cirq developments using the approach that best suits your needs:
+
+*   For releases and major announcements: sign up to the low-volume mailing list
+    [`cirq-announce`].
+*   For releases only:
+    *   Via GitHub notifications: configure [repository notifications] for Cirq.
+    *   Via Atom/RSS from GitHub: subscribe to the GitHub [Cirq releases Atom feed].
+    *   Via RSS from PyPI: subscribe to the [PyPI releases RSS feed] for Cirq.
+
+Cirq releases take place approximately every quarter.
+
+[`cirq-announce`]: https://groups.google.com/forum/#!forum/cirq-announce
+[repository notifications]: https://docs.github.com/github/managing-subscriptions-and-notifications-on-github/configuring-notifications
+[Cirq releases Atom feed]: https://github.com/quantumlib/Cirq/releases.atom
+[PyPI releases RSS feed]: https://pypi.org/rss/project/cirq/releases.xml
+
+### Questions and Discussions
+
+*   Do you have questions about using Cirq? Post them to the [Quantum Computing
+    Stack Exchange] and tag them with the [`cirq`] tag. You can also search past
+    questions using that tag – it's a great way to learn!
+*   Would you like to get more involved in Cirq development? _Cirq Cynq_ is our
+    biweekly virtual meeting of contributors to discuss everything from issues to
+    ongoing efforts, as well as to ask questions. Become a member of
+    [_cirq-dev_](https://groups.google.com/forum/#!forum/cirq-dev) to get an
+    automatic meeting invitation!
+
+[Quantum Computing Stack Exchange]: https://quantumcomputing.stackexchange.com
+[`cirq`]: https://quantumcomputing.stackexchange.com/questions/tagged/cirq
+
+### Issues and Pull Requests
+
+*   Do you have a feature request or want to report a bug? [Open an issue on
+    GitHub] to report it!
+*   Do you have a code contribution? Read our [contribution guidelines], then
+    open a [pull request]!
+
+[Open an issue on GitHub]: https://github.com/quantumlib/Cirq/issues/new/choose
+[contribution guidelines]: https://github.com/quantumlib/cirq/blob/main/CONTRIBUTING.md
+[pull request]: https://help.github.com/articles/about-pull-requests
+
+## Citing Cirq
+
+When publishing articles or otherwise writing about Cirq, please cite the Cirq
+version you use – it will help others reproduce your results. We use Zenodo to
+preserve releases. The following links let you download the bibliographic
+record for the latest stable release of Cirq in various popular formats:
+
+<div align="center">
+
+[![Download BibTeX bibliography record for latest Cirq
+release](https://img.shields.io/badge/Download%20record-e0e0e0.svg?style=flat-square&logo=LaTeX&label=BibTeX&labelColor=106f6e)](https://zenodo.org/records/8161252/export/bibtex)&nbsp;&nbsp;
```

Comment:
This downloads a record for v1.2.0.  Is there a link that redirects to the most recent version?

If not, please replace `8161252` with `11398048` to get v1.4.0 and add a short note to the zenodo section of release.md that these citations need update.

Reply:
Thanks for catching that.

It tooks a little bit of work, but I finally found a way (not via Zenodo) to get BibTeX and CSL format for the latest version given the DOI. No luck with xml, so I'll just drop that and provide only the other two.